cylc/flow/tests/tui/test_util.py

```diff
@@ -160,12 +153,6 @@ def test_compute_tree():
             }
         ],
         'familyProxies': [
-            {  # root family node
-                'name': 'root',
-                'id': '1|root',
-                'cyclePoint': '1',
-                'firstParent': None
-            },
```

Comment:
This was in here to ensure that the root node got filtered if present.

BTW this whole method will need to be re-implemented for deltas.

Reply:
Yup, removed as the data is filtered in the server side in this PR like in Cylc UI. Probably safe, as the root family shouldn't be present any longer I think.